### ISSUE TITLE
[Accessibility]: Main nav dropdown doesn't automatically collapse on page load

### ISSUE THREAD

**Issue by benvenker** [sourcegraph/sourcegraph#35149]
### Audit type

Mobile device navigation

### Problem description

When I click on the Monitoring link from the dropdown, the dropdown remains expanded when the page load

![image](https://user-images.githubusercontent.com/4605817/167482316-70e3dc08-b8eb-4493-9d5b-f495706cbd29.png)

### Expected behavior

I would expect the dropdown to collapse when the Monitoring page loads (or as soon as I click on it and the new page is loading).

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.
